### DISCUSSION TITLE
AttributeError: 'NumpyOps' object has no attribute 'cblas'

### DISCUSSION THREAD

**Discussion by OJcoder17** [explosion/spaCy#11250] [Help: Installation]
## How to reproduce the behaviour
 I am trying to use a tokenizing method wtih the spacy module however I receive an error whenever I try to run it; AttributeError: 'NumpyOps' object has no attribute 'cblas'
![Screenshot (11)](https://user-images.githubusercontent.com/48865894/181861962-23d82045-9a38-4ab6-aaaf-120ab614ad52.png)

Your Environment

* Operating System: Windows 11
* Python Version Used: latest version of python
* spaCy Version Used: spacy-3.4.0
* Environment Information: Jupyter Notebook

**Comment by polm** [comment]
Sorry you're having trouble with that, that's a weird error. 

How did you install spaCy? It seems like some of the dependencies may have been installed incorrectly. I would recommend you follow the instructions in the [Quickstart](https://spacy.io/usage) in a fresh environment. 

Also. please copy and paste code or errors as text, not as screenshots. Also please share the output of `spacy info` so that we know the exact details of your environment.

**Reply by TutubanaS** [reply]
(base) C:\Windows\system32>spacy info
2022-08-27 01:22:00.570216: W tensorflow/stream_executor/platform/default/dso_loader.cc:64] Could not load dynamic library 'cudart64_110.dll'; dlerror: cudart64_110.dll not found
2022-08-27 01:22:00.570369: I tensorflow/stream_executor/cuda/cudart_stub.cc:29] Ignore above cudart dlerror if you do not have a GPU set up on your machine.

============================== Info about spaCy ==============================

spaCy version    3.4.1
Location         C:\Users\Tutu\anaconda3\lib\site-packages\spacy
Platform         Windows-10-10.0.19044-SP0
Python version   3.9.12
Pipelines        en_core_web_sm (3.4.0)


(base) C:\Windows\system32>



Even though I installed the packages again, I still suffer from this issue. Once I create the model and put a string in it like nlp("Hello World!"), I get this error

**Reply by polm** [reply]
Sorry you're having trouble with this. Not sure what's going on, but maybe you're using 32 bit Python? You can check with this command:

```
python -c "import sys;print(\"%x\" % sys.maxsize, sys.maxsize > 2**32)"
```

If it says `False` you are using 32bit Python, which is not supported - I recommend installing 64bit Python. 

If that's not the issue, then could you share the output of `conda list` so we can tell what version of everything is installed? 

You might also try installing things in a blank environment (so not `base`) - `base` is not empty, which causes issues sometimes.
